src/freenas/etc/logrotate.d/netdata_override

```diff
@@ -0,0 +1,8 @@
+/var/log/netdata/*.log {
```

Comment:
Would this override only specific logging configuration netdata package itself provides or is like a complete override of what's specified here and original provided config is retained if that config options are not specified here ?

Reply:
Yes so for example if `abcd` is specified in original config and is not overridden here, that will be retained when the configuration is applied as a whole ✅